internal/service/handlers/form_by_id.go

```diff
@@ -0,0 +1,47 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/rarimo/geo-auth-svc/pkg/auth"
+	"github.com/rarimo/geo-forms-svc/internal/service/requests"
+	"gitlab.com/distributed_lab/ape"
+	"gitlab.com/distributed_lab/ape/problems"
+)
+
+func FormByID(w http.ResponseWriter, r *http.Request) {
```

Comment:
If it's form status and not form, rename all the stuff. Use /status in endpoints instead of /forms, rename handlers and DB handlers. It might be confusing otherwise.

In general, make API more RESTful: if you get form it's /form or /forms, for status it's /status. /forms/last is a good name. /forms/submit is not good, but we are leaving it for backwards compatibility.